src/test/java/com/hazelcast/kubernetes/KubernetesConfigTest.java

```diff
@@ -275,4 +275,32 @@ private static String createTestFile(String expectedContents)
         }
         return temp.getAbsolutePath();
     }
+
+    @Test
+    public void propertyServiceNameIsEmpty() {
+        // given
+        Map<String, Comparable> properties = createProperties();
+        properties.put(SERVICE_NAME.key(), "  ");
+        properties.put(SERVICE_DNS.key(), "service-dns");
+        new KubernetesConfig(properties);
```

Comment:
```suggestion
        // given
        Map<String, Comparable> properties = createProperties();
        properties.put(SERVICE_NAME.key(), "  ");
        properties.put(SERVICE_DNS.key(), "service-dns");

        // when
        KubernetesConfig config = new KubernetesConfig(properties);

        // then
        assertEquals("service-dns", config.getServiceDns());
```

Could you change this test to this format? And could you also change the other tests the same way?